.github/workflows/check-pr.yaml

```diff
@@ -0,0 +1,48 @@
+name: Check PR
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check:
+    runs-on: ubuntu-24.04
```

Comment:
[memo]

I will work on raising it to the latest version of ubuntu in the following Issues, but I used only the latest version of the corresponding `check-pr.yaml` in this PR.

https://github.com/yumemi-inc/flutter-yumemi-lints/issues/164